server/controllers/CommentsController.js

```diff
@@ -0,0 +1,304 @@
+/* eslint-disable class-methods-use-this */
+import { Op } from 'sequelize';
+import db from '../database/models';
+
+const {
+  Article,
+  Comment,
+  User
+} = db;
+
+/**
+ * @class CommentsControllers
+ */
+class CommentsControllers {
+  /**
+   * @description Create a comment for an article
+   *
+   * @param {Object} req - HTTP Request
+   * @param {Object} res - HTTP Response
+   * @param {*} next - Next function
+   *
+   * @return {Object} Returned object
+   */
+  static createComment(req, res, next) {
+    const {
+      body: commentBody
+    } = req.body.comment;
+
+    Article.find({
+      where: {
+        slug: {
+          [Op.eq]: req.params.slug,
+        }
+      }
+    }).then((article) => {
+      if (!article) {
+        return res.status(404).json({
+          message: 'Article does not exist'
+        });
+      }
+
+      Comment.create({
+        articleId: article.id,
+        userId: req.decoded.userId,
+        body: commentBody.trim()
+      }).then((comment) => {
+        User.findById(comment.userId, {
```

Comment:
```
User.findById(comment.userId, {
          attributes: ['username', 'bio', 'image']
        })
```
You are using this frequently to get user details when we can always get user details from the token

Reply:
The token contains just the id of the user.